src/main/java/net/earthcomputer/clientcommands/command/FramerateCommand.java

```diff
@@ -0,0 +1,78 @@
+package net.earthcomputer.clientcommands.command;
+
+import com.mojang.brigadier.CommandDispatcher;
+import net.fabricmc.fabric.api.client.command.v2.FabricClientCommandSource;
+import net.minecraft.client.Minecraft;
+import net.minecraft.network.chat.Component;
+
+import static com.mojang.brigadier.arguments.IntegerArgumentType.*;
+import static net.fabricmc.fabric.api.client.command.v2.ClientCommandManager.*;
+
+public class FramerateCommand {
+    private static final int[] COMMON_REFRESH_RATES = new int[] {
```

Comment:
Perhaps just me but I'd put these on the same line - the list is not that long.

Reply:
they were long enough to run over my 120 char line limit, so...

Reply:
We exceed that limit in 918 other places, I shouldn't worry about it.

Reply:
I do also think it looks better chop'd tho. but I'd also consider this as acceptable codestyle
![image](https://github.com/user-attachments/assets/2e414252-5633-4fa2-b456-528f2d77b74b)